docker/php/php.ini

```diff
@@ -0,0 +1 @@
+date.timezone= "UTC"
```

Comment:
Interesting. That's probably the best configuration. Servers configured in France will probably have Europe/Paris. We need to adapt the code to support many timezone... And this is a critical issue.